projects/python-client/src/datapane/client/api/files.py

```diff
@@ -107,7 +107,7 @@ def write_file(self, f: IO, x: U):
         f.write(self.render_html(x))
 
     def to_block(self, x: T) -> DataBlock:
-        return Table(x) if self._get_cells(x) <= self.TABLE_CELLS_LIMIT else DataTable(x)
```

Comment:
I recall we discussed this on the Basecamp message, however I actually forgot the logic here is configurable - would we want to consider reducing `TABLE_CELLS_LIMIT` instead to, say, 100?

Reply:
Sure - done!